bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/schema/ApiPageParser.java

```diff
@@ -66,6 +74,8 @@
 public class ApiPageParser extends AbstractSimpleMarkupHandler {
 
     private final Logger logger = LoggerFactory.getLogger(ApiPageParser.class);
+    private final String OHM = "" + (char) 0x2126; // U+2126 is the 'real' OHM sign
+    private final String KOHM = "k" + (char) 0x2126; // U+2126 is the 'real' OHM sign
```

Comment:
Not looked into detail, but i see some special handling for ohm and killo ohm. 
According to https://www.openhab.org/docs/concepts/units-of-measurement.html ohm is supported and by default the kilo prefix is also supported. Could you elaborate on why the default UoM handling is insufficient?

Reply:
Your message got me thinking and I think I have the problem: OHM is mistakenly registered with the Greek omega sign (U+03A9) and not with the supposed OHM sign (U+2126). TA reports the correct OHM sign and the lookup fails.
The screenshot shows the Greek omega in the `values` from the unit definition:
![grafik](https://github.com/user-attachments/assets/fe9ec34b-fe67-4850-9dd9-f3b9aebefa35)
And this is how it arrives from TA:
![grafik](https://github.com/user-attachments/assets/2b9b44dd-cec4-4d60-a151-0b9aca351233)
I've already looked into the issues of https://github.com/unitsofmeasurement/indriya/ but it seems nobody noticed it yet.
I could do a
```
                                    // Special rewrite for electrical resistance measurements
                                    // U+2126 is the 'real' OHM sign, but it seems to be registered as Greek Omega
                                    // (U+03A9) in the units
                                    String unitStrRepl = unitStr.replace((char) 0x2126, (char) 0x03A9);
                                    Unit<?> parsedUnit = SimpleUnitFormat.getInstance().parse(unitStrRepl);
```
instead of defining the own units. What do you think?

Reply:
Maybe just perform a char replace before feeding the value to UoM so that you do not need any custom indriya override ?!

Reply:
This is was I suggested with the snipplet above ;) I'll rework this part.

Reply:
As it holds reference to the `SimpleUnitFormat` it seemed like a custom something.

Reply:
I found a solution (by using `QuantityType`) without accessing `SimpleUnitFormat` directly. I have changed it and force-pushed the the commits.